src/zepto.js

```diff
@@ -678,6 +678,8 @@ var Zepto = (function() {
         $this.css(props)
       })
       if (!this.length) return null
+      if (!Zepto.contains(document.documentElement, this[0]))
```

Comment:
My first take on the problem was with a `try...catch` then I remembered that they are expensive.
I'm not familiar with jsperf, but would it be something like this? http://jsperf.com/try-catch-3